third_party/blake3/BUILD.bazel

```diff
@@ -24,13 +25,16 @@ cc_library(
         "c/blake3_dispatch.c",
         "c/blake3_portable.c",
     ] + select({
-        ":windows": [
+        ":x86_64_windows": [
             "c/blake3_avx2_x86-64_windows_msvc.asm",
             "c/blake3_avx512_x86-64_windows_msvc.asm",
             "c/blake3_sse2_x86-64_windows_msvc.asm",
             "c/blake3_sse41_x86-64_windows_msvc.asm",
         ],
-        "//conditions:default": [
+        "@platforms//cpu:arm64": [
+            "c/blake3_neon.c",
```

Comment:
Is your arm64 actually aarch64? If not, according to blake3_neon.c, you have to set  -DBLAKE3_USE_NEON=1 to actually use neon instructions. Should be something like

copts = select({
        "@platforms//cpu:arm64": [ "DBLAKE3_USE_NEON=1"],
        "//conditions:default": [],
        }),

Reply:
It looks like the default value of that macro is detected automatically. From `blake3_impl.h`:
```
#if defined(__aarch64__) || defined(_M_ARM64)
#define IS_AARCH64
#endif
```
and then later:
```
#if !defined(BLAKE3_USE_NEON) 
  // If BLAKE3_USE_NEON not manually set, autodetect based on AArch64ness
  #if defined(IS_AARCH64)
    #define BLAKE3_USE_NEON 1
  #else
    #define BLAKE3_USE_NEON 0
  #endif
#endif
```
I disassembled the binary that I built on my arm64 machine and verified that the NEON version of the function is actually being called:
```
000000000003de20 <blake3_hash_many>:
   3de20:       14000ca8        b       410c0 <blake3_hash_many_neon>
```